composer.json

```diff
@@ -13,23 +13,27 @@
             "email": "pabloelcolombiano@gmail.com"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/lordsimal/cakephp-test-suite-light"
+        }
+    ],
```

Comment:
This will be removed when https://github.com/vierge-noire/cakephp-test-suite-light/pull/61 is merged

Reply:
The PR was merged into next. We will merge it into main and tag once CakePHP 5 officially is released